spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -9,7 +9,7 @@ This project provides support for using Spring Cloud Data Flow with Kubernetes a
 
 == Deploying the Spring Cloud Data Flow server on Kubernetes
 
-In this section we will deploy the Spring Cloud Data Flow Server to a Kubernetes cluster.  We need to make available several Kubernetes services for the Spring Cloud Data Flow server to use. We need an RDBMS service for the app registry and stream and task repositories. We also need a transport option of either Kafka or Rabbit MQ. In addition to this we need a Redis instance if we are planning on using the analytics features.
+In this section we will deploy the Spring Cloud Data Flow Server to a Kubernetes cluster.  We need to make available several services for the Spring Cloud Data Flow server to use. We need an RDBMS service for the app registry and stream and task repositories. We also need a transport option of either Kafka or Rabbit MQ. In addition to this we need a Redis instance if we are planning on using the analytics features.
```

Comment:
> We need to make available several services for the Spring Cloud Data Flow server to use. We need an RDBMS service for the app registry and stream and task repositories. We also need a transport option of either Kafka or Rabbit MQ. In addition to this we need a Redis instance if we are planning on using the analytics features.

How about: 

"Operationalizing Spring Cloud Data Flow depends on few services and its availability. For example, we need an RDBMS service for the app registry, stream, and task repositories. For streaming pipelines, we also need a transport option such as Apache Kafka or Rabbit MQ. In addition to this, we need a Redis service if the analytics features are in use."